pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.poetry]
+name = "tokyo-whisperers"
+version = "0.1.0"
+description = ""
+authors = ["mmbajo <markarjaybajo@gmail.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.10"
+reazonspeech-nemo-asr = {git = "https://github.com/reazon-research/ReazonSpeech", subdirectory = "pkg/nemo-asr"}
```

Comment:
📝 ReazonSpeech models is defined in our environement. Once installed via `poetry install` you can call the nemo model

https://research.reazon.jp/projects/ReazonSpeech/api/reazonspeech.nemo.asr.html#